src/native/thrift/nativeConnection.ts

```diff
@@ -94,7 +89,17 @@ export class NativeConnection<Context = void> extends ThriftConnection {
         const message = this.outBuffer.shift();
         if (message) {
             console.log("Sending next message")
```

Comment:
Should we remove this or change it to `console.debug`?

Reply:
We shouldn't send them to Prod but maybe we look at adding https://stackoverflow.com/questions/41040266/remove-console-logs-with-webpack-uglify